app/models/webauthn_credential.rb

```diff
@@ -18,4 +20,16 @@ def send_creation_email
   def send_deletion_email
     Mailer.webauthn_credential_removed(user_id, nickname, Time.now.utc).deliver_later
   end
+
+  def set_user_mfa_level_create
+    if user.count_webauthn_credentials && user.totp_disabled?
+      user.update!(mfa_level: "ui_and_api")
+    end
+  end
```

Comment:
`user.count_webauthn_credentials` what if the user already has a webauthn device? It looks like the mfa level will be updated.

Reply:
Fixed in this commit: https://github.com/rubygems/rubygems.org/pull/3821/commits/3632776424fbd66bd1c7a54281e1bc4d46fdab87

(check if the credential is the first one)